### PR TITLE
Add Vonage Video Android and iOS SDK 2.26.1 support (with API enhancements)

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

**Review by jvalli — APPROVED** [review]

File: android/src/main/java/com/opentokreactnative/utils/Utils.java [hunk 1]
```diff
@@ -100,6 +101,25 @@ public static List<IceServer> sanitizeIceServer(ReadableArray serverList) {
         return iceServers;
     }
 
+    public static ArrayList<VideoTransformer> sanitizeVideoTransformerList(PublisherKit publisher, ReadableArray transformerList) {
```
Comment (goncalocostamendes) [inline android/src/main/java/com/opentokreactnative/utils/Utils.java:104]: otkit internally does this sort of checks. Trying to set Background Replacament should not lead to anything. If it leads to a crash or something similar, that is a bug on otkit

File: docs/OTSubscriber.md [hunk 2]
```diff
@@ -83,6 +88,7 @@ class App extends Component {
     this.subscriberProperties = {
       subscribeToAudio: false,
       subscribeToVideo: true,
+      subscribeToCaptions: false,
```
Comment (goncalocostamendes) [inline docs/OTSubscriber.md:91]: This value should be true, right? A subscriber will only start receiving captions by calling the subscribeToCaptions(true)

**Review by goncalocostamendes — COMMENTED** [review]
Seems the subscriber is never actually starting the captions

File: docs/OTSubscriber.md (hunk 2 above)
Comment (jeffswartz) [reply docs/OTSubscriber.md:91]: This is just a code sample on how to use OTSubscriber properties. It is not a sample showing how to use captions.

File: docs/OTPublisher.md [hunk 3]
```diff
@@ -159,5 +165,52 @@ A [streamingEvent](./EventData.md#streamingEvent) object is passed into the even
 * **streamDestroyed** (Object) -- Sent when the publisher stops streaming.
 A [streamingEvent](./EventData.md#streamingEvent) object is passed into the event handler.
 
+**setVideoTransformers()** -- Sets video transformers for the publisher. This method has one parameter -- and array of objects defining each transformer to apply to the publisher's stream. A transformer object has two properties:
+
+* `name` (String) -- Either 'backgroundBlur' (for a background blur filter) or 'backgroundImageReplacement' (for a background image replacement filter). Android only supports the 'backgroundBlur' transformer (and it is a beta feature in Android).
```
Comment (goncalocostamendes) [inline docs/OTPublisher.md:170]: "BackgroundBlur" instead of "backgroundBlur"

**Pull request merged**